README.md

```diff
@@ -1 +1,88 @@
-# SwarmForm
\ No newline at end of file
+# SwarmForm
+
+SwarmForm is a distributed workflow management system for High Performance Computing environments. SwarmForm introduces task clustering to reduce the makespan of workflows executed using it by minimizing the overheads. SwarmForm follows a distributed architecture with independent programs controlling different functional layers in workflow management.
+## Getting Started
```

Comment:
Need an empty line to separate the paragraph

Reply:
These are not being shown on the .md file. But better to add empty lines where necessary and remove the extra empty lines in different sections